README.md

```diff
@@ -20,9 +20,61 @@ Features:
 ## v6.0.0 migration
 
 ### Silence the warnings
+
 In version 5 there are many warnings to mark deprecated feature/functions/methods.
 They can be silenced by setting the `GSPREAD_SILENCE_WARNINGS` environment variable to `1`
 
+### HTTP Client
+
+HTTP Clients have moved a dedicated file. GSpread uses by default the `HTTPClient`.
+If you wish to use the new `BackoffHTTPClient` please update your code as follow:
```

Comment:
```suggestion
HTTP Clients have moved to a dedicated file. By default, gspread uses `HTTPClient`.
Also provided is a new `BackoffHTTPClient`, which retries failed requests with exponential time delay. It can be used with:
```